lib/chartmogul/customer.js

```diff
@@ -56,6 +56,8 @@ Customer.modify = Resource._method('PATCH', '/v1/customers/{customerUuid}');
 
 Customer.merge = Resource._method('POST', '/v1/customers/merges');
 
+Customer.unmerge = Resource._method('POST', '/v1/customers/unmerges');
```

Comment:
I just realized we aren't documenting these methods in the README files of the SDKs. Is it intentional to not update the README, something like a silent release?

Reply:
Ah -- nope, just an oversight. : ) Will fix!